src/content/add-to-app/android/project-setup.md

```diff
@@ -394,10 +414,25 @@ To change the name of this project, set
 Include this project in the host app's `settings.gradle` file.
 :::
 
+#### Updating `settings.gradle`
+
 Include the Flutter module as a subproject in the host app's
 `settings.gradle`. This example assumes `flutter_module` and `MyApp`
 exist in the same directory
 
+If you are using Kotlin, apply the following changes:
+
+```kotlin title="MyApp/settings.gradle.kts"
+// Include the host app project. Assumed existing content.
+include(":app")            
+// Replace "flutter_module" with whatever package_name you supplied when you ran:
+// `$ flutter create -t module [package_name]
+val filePath = settingsDir.parentFile.toString() + "/flutter_module/.android/include_flutter.groovy"
+apply(from = File(filePath))
+```
+
+If you are using Groovy, apply the following changes:
+
```

Comment:
It would be great to modify the groovy suggestion to use the new `apply from` way here, so

```groovy
def filePath = settingsDir.parentFile.toString() + "/flutter_module/.android/include_flutter.groovy"
apply from: filePath
```

Reply:
Done!